v3/internal/runtime/desktop/@wailsio/runtime/src/cancellable.ts

```diff
@@ -841,7 +841,7 @@ function rejectorFor<T>(promise: CancellablePromiseWithResolvers<T>, state: Canc
  * Returns a promise that fulfills once all cancellation procedures for the given values have settled.
  */
 function cancelAll(parent: CancellablePromise<unknown>, values: any[], cause?: any): Promise<void> {
-    const results = [];
+    const results: Promise<any>[] = [];
```

Comment:
This was throwing an error in the `lit-ts` project without this.  I'm not sure why it didn't error in the runtime project too.